src/screens/TopicMemberScreen/index.js

```diff
@@ -35,7 +35,7 @@ const TopicMemberScreen = () => {
   const [topicDataFollowedUsers, setTopicDataFollowedUsers] = React.useState([]);
   const [topicDataUnfollowedUsers, setTopicDataUnfollowedUsers] = React.useState([]);
   const [isFocus, setIsFocus] = React.useState(true);
-  const [isFirstTimeOpen, setIsFirstTimeOpen] = React.useState(true);
+  const [isFirstTimeOpen, setIsFirstTimeOpen] = React.useState(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The initial state of `isFirstTimeOpen` has been changed from `true` to `false`. This could potentially alter the component's behavior at initialization. Make sure this change aligns with the expected behavior of your application.
```diff
- const [isFirstTimeOpen, setIsFirstTimeOpen] = React.useState(true);
+ const [isFirstTimeOpen, setIsFirstTimeOpen] = React.useState(false);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->